src/components/RobotPositionDisplay.tsx

```diff
@@ -30,6 +32,10 @@ export class RobotPositionDisplay extends React.Component<Props> {
           <h3>Y:</h3>
           <NumberDisplay value={Math.round(this.props.y)} onChange={(val) => this.props.yChanged(val)} width={5} />
         </section>
+        <section style={{ textAlign: 'center' }}>
+          <h3>Z:</h3>
+          <NumberDisplay value={Math.round(this.props.z)} onChange={(val) => this.props.zChanged(val)} width={5} />
```

Comment:
In the position input boxes, if you delete the current number, it changes to `NaN` and is tricky to get it back to a number. I'm guessing this was an issue before.

Reply:
Yes that is probably a separate issue that needs to be fixed. Since everything changes based on set now, you can always delete NaN and replace it with a number.